aspnet/publishing/iis.rst

```diff
@@ -54,6 +54,27 @@ For more information on the ASP.NET Core Module, including configuration of the
 Application configuration
 -------------------------
 
+Enabling the `IISIntegration` service
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Ensure that the `UseIISIntegration()` is injected:
+
+.. code-block:: csharp
+
+  var host = new WebHostBuilder()
+    .UseKestrel()
+    .UseIISIntegration()
+    .UseStartup<Startup>()
+    .Build();
```

Comment:
It's up to @Rick-Anderson and @danroth27, but I hope for consistency across all of them with https://github.com/aspnet/cli-samples